AsyncDisplayKit/ASDisplayNode.h

```diff
@@ -631,21 +638,23 @@ extern NSInteger const ASDefaultDrawingPriority;
 @interface ASDisplayNode (UIViewBridge)
 
 /**
- * Marks the view as needing display. Convenience for use whether the view / layer is loaded or not. Safe to call from a background thread.
+ * Marks the view as needing display. Convenience for use whether the view / layer is loaded or not. Safe to call
+ * from a background thread.
  */
 - (void)setNeedsDisplay;
 
 /**
  * Marks the node as needing layout. Convenience for use whether the view / layer is loaded or not. Safe to call from a background thread.
- * 
- * If this node was measured, calling this method triggers an internal relayout: the calculated layout is invalidated,
- * and the supernode is notified or (if this node is the root one) a full measurement pass is executed using the old constrained size.
- *
- * Note: ASCellNode has special behavior in that calling this method will automatically notify 
- * the containing ASTableView / ASCollectionView that the cell should be resized, if necessary.
  */
 - (void)setNeedsLayout;
 
+/**
+ * Recalculate the receiver’s layout, if required.
```

Comment:
This part of comment is incorrect - it doesn't recalculate unless the current one is invalid.

I think there are some challenges with the definition of this method and it's possible we may not want to name it the same as UIKit (similar to layoutThatFits / not having sizeToFit because ASDK is more flexible).

Are you open to putting this in +Beta for now so it can be used but no promise that it's behavior won't change / discourage adoption by requiring manual header import?